cognite/client/_api/datapoints.py

```diff
@@ -322,6 +322,29 @@ def insert_multiple(self, datapoints: List[Dict[str, Union[str, int, List]]]) ->
         dps_poster = DatapointsPoster(self)
         dps_poster.insert(datapoints)
 
+    def delete_point(self, timestamp: Union[int, str, datetime], id: int = None, external_id: str = None) -> None:
+        """Delete a range of datapoints from a time series.
+
+        Args:
+            timestamp (Union[int, str, datetime]): The timestamp of the datapoint to delete
+            id (int): Id of time series to delete data from
+            external_id (str): External id of time series to delete data from
+
+        Returns:
+            None
+
+        Examples:
+
+            Deleting the last week of data from a time series::
+
+                >>> from cognite.client import CogniteClient
+                >>> c = CogniteClient()
+                >>> c.datapoints.delete_range(start="1w-ago", end="now", id=1)
```

Comment:
```suggestion
                >>> c.datapoints.delete_point(timestamp="2000-01-01T00:00:00Z", id=1)
```
...assuming that the timestamp format is accepted, I haven't tested it. =)